### PR TITLE
Master recruitement aeb

### PR DESCRIPTION
**Pull request by awa-odoo**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/hr_recruitment/views/hr_job_views.xml [hunk 1]
```diff
@@ -32,12 +42,12 @@
                                     <div class="o_primary col-12">
                                         <span><t t-esc="record.name.value"/></span>
                                     </div>
-                                    <div class="o_kanban_record_subtitle col-12 text-muted">
+                                    <div class="o_kanban_record_subtitle text-muted">
                                         <field name="user_id" />
                                     </div>
                                     <field name="is_favorite" widget="boolean_favorite" nolabel="1"/>
-                                    <div t-if="record.alias_name.value and record.alias_domain.value and record.state.raw_value == 'recruit'" class="o_secondary o_job_alias">
```
Comment (awa-odoo) [inline addons/hr_recruitment/views/hr_job_views.xml]: Why did you remove the t-if here?

File: addons/hr_recruitment/models/hr_job.py [hunk 2]
```diff
@@ -78,6 +78,18 @@ def _compute_application_count(self):
...
+        print(self)
+        return {
+            'name': _('Job Position'),
+            'type': 'ir.actions.act_window',
+            'view_mode': 'form',
+            'res_model': 'hr.job',
+            'res_id': self.id,
+        }
+        # action = self.with_context(active_id=self.id).env.ref('hr_recruitment.act_jobs_form_single').read()[0]
+        # return action
```
Comment (awa-odoo) [inline addons/hr_recruitment/models/hr_job.py]: Don't keep commented out code.

File: addons/hr_recruitment/views/hr_recruitment_views.xml [hunk 3]
```diff
@@ -25,24 +25,20 @@
         <field name="model">hr.applicant</field>
         <field name="arch" type="xml">
             <tree string="Applicants" decoration-bf="message_needaction==True" multi_edit="1">
-                <field name="message_needaction" invisible="1"/>
```
Comment (awa-odoo) [inline addons/hr_recruitment/views/hr_recruitment_views.xml]: Removing this field will break the` decoration-bf="message_needaction==True"`

File: addons/hr_recruitment/views/hr_recruitment_views.xml [hunk 4]
```diff
@@ -25,24 +25,20 @@
...
-                <field name="company_id" groups="base.group_multi_company" readonly="1"/>
+                <field name="create_date" optional="show"/>
+                <field name="priority" optional="show"/>
+                <field name="salary_expected" optional="show"/>
+                <field name="availability" optional="show"/>
+                <field name="salary_proposed" optional="hidden"/>
+                <field name="email_from" optional="hidden"/>
+                <field name="partner_phone" optional="hidden"/>
+                <field name="medium_id" optional="hidden"/>
+                <field name="source_id" optional="hidden"/>
```
Comment (awa-odoo) [inline addons/hr_recruitment/views/hr_recruitment_views.xml]: Why did you remove the readonly modifier on some of those fields?

File: addons/hr_recruitment/views/hr_recruitment_views.xml [hunk 5]
```diff
@@ -878,14 +874,15 @@ action = act
     <record id="hr_applicant_action_from_department" model="ir.actions.act_window">
         <field name="name">New Applications</field>
         <field name="res_model">hr.applicant</field>
-        <field name="view_mode">kanban,tree,form,graph,calendar,pivot</field>
+        <field name="view_mode">kanban,tree,form,graph,calendar,pivot,list</field>
```
Comment (awa-odoo) [inline addons/hr_recruitment/views/hr_recruitment_views.xml]: view_mode "list" ? 
Don't even know what that is.
Looks like an old compatibility layer for "tree".

Why did you add that?

File: addons/hr_recruitment/views/hr_recruitment_views.xml [hunk 6]
```diff
@@ -878,14 +874,15 @@ action = act
...
         <field name="context">{
             'search_default_department_id': active_id,
             'default_department_id': active_id}
         </field>
         <field name="domain">[('stage_id.sequence','&lt;=','1')]</field>
     </record>
 
+
```
Comment (awa-odoo) [inline addons/hr_recruitment/views/hr_recruitment_views.xml]: Unnecessary extra space.

File: addons/utm/data/utm_data.xml [hunk 7]
```diff
@@ -34,6 +34,9 @@
         <record model="utm.source" id="utm_source_twitter">
             <field name="name">Twitter</field>
         </record>
+        <record model="utm.source" id="utm_source_linkdin">
```
Comment (MiquelRForgeFlow) [inline addons/utm/data/utm_data.xml]:
```suggestion
        <record model="utm.source" id="utm_source_linkedin">
```

File: addons/utm/data/utm_data.xml [hunk 8]
```diff
@@ -34,6 +34,9 @@
...
+            <field name="name">Linkdin</field>
```
Comment (MiquelRForgeFlow) [inline addons/utm/data/utm_data.xml]:
```suggestion
            <field name="name">LinkedIn</field>
```
as this is `master` branch, shouldn't we try to fix this kind of typo?

File: addons/hr_recruitment/views/hr_recruitment_views.xml (hunk 5 above)
Comment (lextjes) [reply addons/hr_recruitment/views/hr_recruitment_views.xml]: I did it by accident , i did removed the entire kanban to test something when , I did put back i did the error maybe an copy paste of another kanban

File: addons/hr_recruitment/views/hr_job_views.xml (hunk 1 above)
Comment (lextjes) [reply addons/hr_recruitment/views/hr_job_views.xml]: I've removed it for testing cause I needed to verify that the email and responsible name are aligned and have same size and typo. But I did put it back in a later commit normally

File: addons/hr_recruitment/views/hr_recruitment_views.xml (hunk 3 above)
Comment (lextjes) [reply addons/hr_recruitment/views/hr_recruitment_views.xml]: Same as for the other remove I did remove it for testing purpose and I did put it back in a later Commit

File: addons/hr_recruitment/views/hr_recruitment_views.xml (hunk 4 above)
Comment (lextjes) [reply addons/hr_recruitment/views/hr_recruitment_views.xml]: When I made the new view tree I forgot to put the readonly attributes on some fields

**Comment by tde-banana-odoo** [comment]
Overseeded by https://github.com/odoo/odoo/pull/47707
